packages/@ourworldindata/grapher/src/entitySelector/EntitySelector.tsx

```diff
@@ -60,162 +56,377 @@ export class EntitySelector extends React.Component<{
+                                    checked={this.isEntitySelected(name)}
+                                    onChange={() => this.onChange(name)}
+                                />
+                            </li>
+                        </Flipped>
+                    ))}
+                </ul>
+            </Flipper>
+        )
+    }
+
+    private renderFooter(): JSX.Element {
+        const { numSelectedEntities, selectedEntityNames } = this.selectionArray
+        const { partitionedVisibleEntities: visibleEntities } = this
+
+        return (
+            <div className="entity-selector__footer">
+                {this.isMultiMode ? (
+                    <>
+                        <div className="footer__selected">
+                            {numSelectedEntities > 0
+                                ? `${numSelectedEntities} selected`
+                                : "Empty selection"}
+                        </div>
                         <button
-                            className="clearSelection"
+                            type="button"
                             onClick={this.onClear}
+                            disabled={visibleEntities.selected.length === 0}
                         >
-                            Clear selection
+                            Clear
                         </button>
-                    ) : undefined}
-                </div>
-
-                <div className="entities">
-                    <div className="searchResults">
-                        {searchResults.length > 0 ? (
-                            <ul>
-                                {searchResults.map((result) => (
-                                    <EntitySearchResult
-                                        key={result.name}
-                                        result={result}
-                                        isMulti={this.isMulti}
-                                        isChecked={selectionArray.selectedSet.has(
-                                            result.name
-                                        )}
-                                        onSelect={this.onSelect}
-                                    />
-                                ))}
-                            </ul>
-                        ) : (
-                            <div className="empty">
-                                {this.manager.entitiesAreCountryLike &&
-                                isCountryName(this.searchInput)
-                                    ? "There is no data for the country, region or group you are looking for."
-                                    : "Nothing turned up. You may want to try using different keywords or checking for typos."}
-                            </div>
-                        )}
+                    </>
+                ) : (
+                    <div className="footer__selected footer__selected--no-wrap">
+                        Current selection: {selectedEntityNames[0]}
```

Comment:
I don't know when that would be the case, but this can probably be undefined in some cases.